docs/manual/macros.md

```diff
@@ -43,6 +43,15 @@ macros are available:
 | %{-f*} |	if present at invocation, the argument to the last occurence of flag f
 | %1, %2, ...|	the arguments themselves (after getopt(3) processing)
 
+With rpm >= 4.17 and disabled option processing the mentioned macros are defined as:
+
+| Macro  | Description | Notes
+| ------ | ----------- | ------
+| %0     | the name of the macro being invoked
+| %*, %** | all arguments
+| %#     | the number of arguments
+| %1, %2, ...|	the arguments themselves
```

Comment:
I'd rather split the option/flag stuff out from the original table to a new one, and explain the logic ("unless option processing is disabled, the following macros are defined for flags that were present..."). That avoids having to repeat the same info twice.

Reply:
I separated that because these macros do have a different meaning 

- enabled option processing

| Macro  | Description
| ------ | -----------
| %*	 | all arguments (unlike shell, not including any processed flags)
| %**	 | all arguments (including any processed flags)
| %1, %2, ...|	the arguments themselves (after getopt(3) processing)

- disabled option processing

| Macro  | Description
| ------ | -----------
| %*, %** | all flags and arguments
| %1, %2, ...| the flags and arguments themselves

How to handle them ?

Reply:
Okay that is a fair point, trying to explain the difference in the description might end up far more confusing than having a separate table for each case. Objecttion withdrawn :smile:

Reply:
Thanks for the patch!